module/Crud/src/Form/AddForm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
```

Comment:
Please add blank line before declare statement 
We use PSR-12 
Refer https://www.php-fig.org/psr/psr-12/